tests/integration-tests/utils.py

```diff
@@ -244,3 +244,14 @@ def paginate_boto3(method, **kwargs):
     for page in paginator.paginate(**kwargs).result_key_iters():
         for result in page:
             yield result
+
+
+def get_username_for_os(os):
+    usernames = {
+        "alinux": "ec2-user",
+        "centos6": "centos",
+        "centos7": "centos",
+        "ubuntu1604": "ubuntu",
+        "ubuntu1804": "ubuntu",
+    }
+    return usernames[os]
```

Comment:
Minor: I'd use `usernames.get(os)`.
With `usernames[os]`, if you pass unsupported OS you'll obtain a `KeyError`, instead with `get()` you'll have `None` as result or it's possible to add a default returned value.